_layouts/default.html

```diff
@@ -5,7 +5,7 @@
   <body>
 
     <div>
-      <a href="{{ site.baseurl }}">
+      <a href="{{ site.baseurl }}/">
```

Comment:
Seems like something's wrong with the `baseurl` if you need to add a trailing slash. If it's the empty string, then you need to set it to `/`.